.travis.yml

```diff
@@ -1,39 +1,36 @@
 language: php
+
 php:
   - 7.0
 
-sudo: false
-
-env:
-  global:
-    - SIMPLETEST_DB=mysql://root:@localhost/journal_cms
-    - SIMPLETEST_BASE_URL="http://127.0.0.1:8080"
-
-services:
-  - mysql
+mysql:
+  database: journal_cms
+  username: root
+  encoding: utf8
 
 before_install:
-  - curl https://puli.io/installer | php
   - echo 'sendmail_path = /bin/true' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer --verbose self-update
   - composer --version
 
 install:
-  - mysql --execute='CREATE DATABASE journal_cms;'
   # We require coveralls like this as it won't work straight from composer.json.
   - composer require satooshi/php-coveralls
   - composer --verbose install
-  - composer drupal-scaffold
```

Comment:
Does that mean we could do `composer install` too in https://github.com/elifesciences/journal-cms-formula/blob/master/salt/journal-cms/init.sls#L51 instead of `composer update`?